jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile

```diff
@@ -17,15 +17,31 @@ pipeline {
 
         stage('Clone Operator Repo') {
             steps {
-                //ToDo Clone the integeratly-operator repo
-                echo "Clone Operator Repo"
+                cleanWs()
+                git 'https://github.com/integr8ly/integreatly-operator'
```

Comment:
We need to be able to specify the branch here. Will eventually be 'master', but should be set to "deloeran-ews" for the time being.

The stages after this will also need access to the base branch so we need to deal with that also.

Reply:
Does this clone the repo and change the directory into it?

Reply:
It does clone and change into the newly cloned directory.

I don't know if you can specify a branch here, I think we'd have to `sh git checkout` but I can add that so we start at the branch we want, it defaults to master.

Reply:
Ignore me, it's really easy, change pushed